Doc/whatsnew/3.14.rst

```diff
@@ -1390,6 +1390,12 @@ urllib
   :func:`~urllib.request.build_opener`.
   (Contributed by Barney Gale in :gh:`84850`.)
 
+uuid
```

Comment:
The `uuid` module should already exist under "Improved modules". This section is for deprecated modules.

Reply:
Err, my bad. I've fixed this.